stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3549,6 +3549,23 @@ final internal class _SwiftDeferredNS${Self}<${TypeParametersDecl}>
   ) {
     bridgedAllKeysAndValues(objects, keys)
   }
+
+  @objc(enumerateKeysAndObjectsWithOptions:usingBlock:)
+  internal func enumerateKeysAndObjects(options: Int, 
+    using block: @convention(block) (Unmanaged<AnyObject>, Unmanaged<AnyObject>,
```

Comment:
I *believe* you can leave out the `Unmanaged` here; it's understood that blocks take arguments at +0.